README.md

```diff
@@ -51,6 +51,24 @@ To run directly from the command line, use `pipenv run`:
 pipenv run python -m updater [-c FILE]
 ```
 
+Alternatively you can install the python module and have a command named `wow-addon-updater` available:
+```bash
+pipenv install
+pipenv shell
+# Make sure you're in a virtual environment! pipenv shell should place you in the venv
+# created by `pipenv install` but to be sure you can check with `which python` and
+# verify that it's $VIRTUAL_ENV/bin/python
+
+# Using the distutils method
+python setup.py install
+# Using pip
+pip install .
```

Comment:
I'll admit I was blindly spitting these lines into my terminal, and hesitated when I ran `python setup.py install` and `pip install .` back to back and it failed. Maybe clarify this instruction with something like

```suggestion
# Either using the distutils method
python setup.py install
# Or using pip
pip install .
```